### PR TITLE
Model of Communication between Agents

### PR DESCRIPTION
**Pull request by mrnolte**
This PR introduces a number of new classes, properties and axioms to model the communication between agents.
It does not yet include qualities of communicated information, e.g., confidence in an answer, but only the general structure of a Communication Action and Communication Tasks with Messages, Senders, Receivers and Channels including a basic cause/reaction relationship between Actions (without any assumptions on perlocutionary effects / BDI).

### CODE REVIEW

File: owl/SOMA-ACT.owl [hunk 1]
```diff
@@ -1028,6 +1317,26 @@ Note that buying an object is NOT PhysicalAcquiring. Buying, or ownership transf
...
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#Message"/>
+        <rdfs:comment xml:lang="en">A role played by some Information Realization that carries meaning, where this meaning is a query of some sort.</rdfs:comment>
+        <rdfs:label xml:lang="en">Query</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/SOMA.owl#QueryingTask -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA.owl#QueryingTask">
```
Comment (mpomarlan) [inline owl/SOMA-ACT.owl:1332]: Shouldn't there be a restriction here saying that a querying task will have a query role?

File: owl/SOMA-ACT.owl [hunk 2]
```diff
@@ -204,6 +308,53 @@ Also defined here are execution state regions, i.e. flags with which to label th
     
 
 
+    <!-- http://www.ease-crc.org/ont/SOMA-ACT.owl#AnsweringTask -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA-ACT.owl#AnsweringTask">
```
Comment (mpomarlan) [inline owl/SOMA-ACT.owl:313]: Shouldn't this task have a role for the answer message?

File: owl/SOMA-SAY.owl [hunk 3]
```diff
@@ -171,8 +159,7 @@ In particular, it defines concepts related to linguistic acts and structures of
 
     <!-- http://www.ease-crc.org/ont/SOMA.owl#AssertionTask -->
 
-    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA.owl#AssertionTask">
-        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#IllocutionaryTask"/>
+    <rdf:Description rdf:about="http://www.ease-crc.org/ont/SOMA.owl#AssertionTask">
```
Comment (mpomarlan) [inline owl/SOMA-SAY.owl:162]: It seems like some axioms about assertiontask got into soma-act while others remained in soma-say. Is there a reason for this or should this be refactored?

File: owl/SOMA-SAY.owl [hunk 4]
```diff
@@ -242,10 +227,9 @@ Let xA, xP be objects filling the agent, patient roles of this schema. Then one
     
 
 
-    <!-- http://www.ease-crc.org/ont/SOMA.owl#CommandTask -->
+    <!-- http://www.ease-crc.org/ont/SOMA.owl#CommandingTask -->
```
Comment (mpomarlan) [inline owl/SOMA-SAY.owl:230]: As before, axioms about CommandingTask got split between soma-act and soma-say.

File: owl/SOMA-SAY.owl [hunk 5]
```diff
@@ -645,10 +585,9 @@ Let xL, xR be objects filling the locatum, relatum roles of this schema. Then on
     
 
 
-    <!-- http://www.ease-crc.org/ont/SOMA.owl#QueryTask -->
+    <!-- http://www.ease-crc.org/ont/SOMA.owl#QueryingTask -->
 
-    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA.owl#QueryTask">
-        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#IllocutionaryTask"/>
+    <rdf:Description rdf:about="http://www.ease-crc.org/ont/SOMA.owl#QueryingTask">
```
Comment (mpomarlan) [inline owl/SOMA-SAY.owl:590]: As before, axioms for QueryingTask got split up between soma-act and soma-say.

File: owl/SOMA-SAY.owl (hunk 3 above)
Comment (mrnolte) [reply owl/SOMA-SAY.owl:162]: Yes, this is intended: SOMA-SAY declares concepts like DeclarativeCause, but SOMA-ACT does not.

File: owl/SOMA-SAY.owl (hunk 4 above)
Comment (mrnolte) [reply owl/SOMA-SAY.owl:230]: See my comment above on assertiontask

File: owl/SOMA-SAY.owl (hunk 5 above)
Comment (mrnolte) [reply owl/SOMA-SAY.owl:590]: See my comment above on assertiontask.

File: owl/SOMA-ACT.owl (hunk 1 above)
Comment (mrnolte) [reply owl/SOMA-ACT.owl:1358]: Added!

File: owl/SOMA-ACT.owl (hunk 2 above)
Comment (mrnolte) [reply owl/SOMA-ACT.owl:313]: Added!

**Review by mpomarlan — APPROVED** [review]

**Pull request merged**